lib/Razor2/String.pm

```diff
@@ -65,15 +69,8 @@ sub hmac2_sha1 {
     return unless $text && $iv1 && $iv2;
     die "no ref's allowed" if ref($text);
 
-    my $ctx = Digest::SHA1->new;
-    $ctx->add($iv2);
-    $ctx->add($text);
-    my $digest = $ctx->hexdigest;
-
-    $ctx = Digest::SHA1->new;
-    $ctx->add($iv1);
-    $ctx->add($digest);
-    $digest = $ctx->hexdigest;
+    my $digest = sha1_hex($iv2, $text);
+    $digest = sha1_hex($iv1, $digest);
```

Comment:
IMO this is a bit confusing due to the reuse of `$digest`. I'd either use `$digest1` and $digest2` or just make that a `sha1_hex($iv1, sha1_hex($iv2, $text))`